project/Rat.scala

```diff
@@ -30,8 +30,10 @@ object Rat {
       * well.
       */
     file("language/dfdl.json"),
+    file("language/tdml.json"),
     file("language/syntax.json"),
     file("language/syntaxes/dfdl.tmLanguage.json"),
+    file("language/syntaxes/tdml.tmLanguage.json"),
     file("package.json"),
     file("svelte/package.json"),
     file(".prettierrc"),
```

Comment:
@NolanMatt Add `src/tdmlEditor/media/codicon.ttf` to this file, this is the binary file causing your Rat Check to fail

Reply:
@stevedlawrence How do I go about labeling it?

Reply:
In addition to updating the appropriate LICENSE/NOTICE fie, we add a mention in in the license section of the README.md. Or alternatively, we try to use ALv2 or other Category A licensed files instead and avoid the issue entirely.

Reply:
Looks like the `codeicon.tff` was removed so I am going to mark this resolved.